doc/manual/cookbooks/shell_simple_2d_smoothing/shell_simple_2d_smoothing_piece.prm

```diff
@@ -0,0 +1,20 @@
+subsection Discretization
+  set Temperature polynomial degree = 2
+
+  subsection Stabilization parameters
+    set Use artificial viscosity smoothing = true
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 1
+end
+
+subsection Compositional initial conditions
+  set Model name = function
+
+  subsection Function
+    set Variable names      = x,y
+    set Function expression = if(sqrt(x*x+y*y)<4000000,1,0)
+  end
+end
```

Comment:
Add a newline here.

This file is a fragment of the complete input file which you've also copied in this directory. I've typically found that showing complete input files in cookbooks is often more distracting than helpful because it's not easy to see (as a reader) what the important parts are. Consequently, I've often come to just show the important snippets, like you do here. If you have no intention of showing the *complete* input file, then you also don't need to add it to the current directory.